src/test/java/uk/nhs/digital/nhsconnect/lab/results/inbound/EdifactParserTest.java

```diff
@@ -0,0 +1,163 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Interchange;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.InterchangeFactory;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.ToEdifactParsingException;
+
+import java.util.List;
+
+@ExtendWith(MockitoExtension.class)
+class EdifactParserTest {
+    private static final String EDIFACT_HEADER = "UNB+UNOA:2+TES5+XX11+020114:1619+00000003";
```

Comment:
Can we move these constants to a test fixture class, and then import it in here?
It's good to separate the test data from the actual tests (easier to read).

So for example, you can create a new package/folder under `src/test/java/uk/nhs/digital/nhsconnect/lab/results` called `fixtures`, and create a class there called `EdifactFixtures`.
And in there, you can do
```
package uk.nhs.digital.nhsconnect.lab.results.fixtures;

import java.util.List;

public class EdifactFixtures {
    public static final String EDIFACT_HEADER = "UNB+UNOA:2+TES5+XX11+020114:1619+00000003";
    public static final String EDIFACT_TRAILER = "UNZ+1+00000003";

    public static final List<String> SAMPLE_EDIFACT = List.of(
            EDIFACT_HEADER + "'",
            ...
            EDIFACT_TRAILER + "'"
    );

    public static final List<String> TRAILER_BEFORE_HEADER_EDIFACT = List.of(
            EDIFACT_HEADER + "'",
            ...
            EDIFACT_TRAILER + "'"
    );


    public static final List<String> MISMATCH_MESSAGE_TRAILER_HEADER_EDIFACT = List.of(
            EDIFACT_HEADER + "'",
            ...
            EDIFACT_TRAILER + "'"
    );
}
```

Reply:
Completed in 403b23c